docs/02-architecture/02-fronts-architecture.md

```diff
@@ -1,69 +1,110 @@
-# Fronts Architecture
+# Fronts architecture
 
 ![Fronts architecture](images/fronts-archirecture.png)
 
-The goal of the Fronts architecture is serve each front via a _single_ blocking call to a persitence layer.
+Rendering a front requires fetching a lot of information from multiple sources (CAPI, Fronts config, Fronts collections). In order to avoid doing this gruelling work inline when a user request is received, all data necessary to render a front is prefetched and aggregated in a single location (S3). 
 
-The system produces static aggregation files, offline, one for each front (`pressed.json`). Each file contains all the content required for one front; it is a complete snapshot of its required ContentApi data, structured by collection, and overlaid by the temporary fronts-only article metdata. Each fulfills the single call made by the Facia app in order to service a request for that front.
+In a nutshell:
+- [Facia press](#facia-press): for each front, fetch all necessary data, aggregate it and store it in S3
+- [Facia](#facia): read the aggregated data from S3 and render the front
 
-The global list of defined fronts - and the definitions of the collections on those fronts - is held in a single configuration file (`config.json`). This file is edited using the Configuration Tool. It is polled and referred to by each component in the architecture. 
+## Facia Press
+It is not a user facing service, it processes events published to queues.
+There are 2 queues:
+- one queue populated by a cron job running in the Admin app.
+- one queue where the Fronts tool (fronts.gutools.co.uk) publishes events when a front is being updated.
+The cron job is very important since content for a front can be coming from CAPI and change without any update from the Fronts tool.
```

Comment:
Perhaps mention the different pressing priorities of editorial, commercial and training pages?